docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -310,89 +414,107 @@ muteVid.addEventListener("click", () => {
 
 </Tab>
 
-We fetch the current state of the local audio and video and toggle them whenever the buttons are clicked.
+<Tab id="mute-unmute-2">
+
+```css:styles.css section=MutingUnmutingLocalTracks sectionIndex=1 tab=CSS
+  .control-bar {
+    display: flex;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    padding: 15px;
+    justify-content: center;
+    z-index: 10;
+  }
+
+  .control-bar > *:not(:first-child) {
+    margin-left: 8px;
+  }
 
-If you try it out now, you may notice that turning off the video makes the tile go blank but turning it on again doesn't bring it back.
-This is due to a change in video track requiring a call to attach again. To fix it let's
-call our `renderPeers` function whenever we change the state of the local video.
+  .btn-control {
+    font-size: 12px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    border: 2px solid #37474f;
+    width: 64px;
+    height: 64px;
+    border-radius: 50%;
+    text-align: center;
+    background-color: #607d8b;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.4);
+    color: white;
+  }
+```
 
-```jsx{8}:script.js section=MutingUnmutingLocalTracks sectionIndex=2
-muteVid.addEventListener("click", () => {
-  const videoEnabled = hmsStore.getState(selectIsLocalVideoEnabled);
+</Tab>
 
-  hmsActions.setLocalVideoEnabled(!videoEnabled);
+### Changing UI based on connection state
 
-  muteVid.textContent = !videoEnabled ? "Hide" : "Unhide";
+Right now, our join form will show up even after we have joined the room. We need a way to know the connection state of the room and hide the form after we've connected.
 
-  renderPeers(hmsStore.getState(selectPeers));
-});
+We can do this by subscribing to the store with the [selectIsConnectedToRoom](/api-reference/javascript/v2/home/content#select-is-connected-to-room) selector from `hmsStore`.
+
+```jsx:index.js section=ChangingUIBasedOnConnectionState sectionIndex=1
+// Showing the required elements on connection/disconnection
+function onConnection(isConnected) {
+  if (isConnected) {
+    form.classList.add("hide");
+    conference.classList.remove("hide");
+    leaveBtn.classList.remove("hide");
+    controls.classList.remove("hide");
+  } else {
+    form.classList.remove("hide");
+    conference.classList.add("hide");
+    leaveBtn.classList.add("hide");
+    controls.classList.add("hide");
+  }
+}
+
+// listen to the connection state
+hmsStore.subscribe(onConnection, selectIsConnectedToRoom);
 ```
 
-![Mute/Unmute](/guides/mute-unmute.gif)
+## Test the app
 
-## Mirroring local video
+### Start the ParcelJS development server
```

Comment:
Is it intentional we are mentioning ParcelJS everywhere? we can simply mention start the development server.

Reply:
done